### PR TITLE
Fix repro build

### PR DESCRIPTION
**Pull request by scgbckbone**

### CODE REVIEW

**Comment by scgbckbone** [comment]
got succes for MK

File: stm32/Makefile [hunk 1]
```diff
@@ -16,6 +16,6 @@
 
 release rc1 clean clobber repro:
 	$(MAKE) -f MK4-Makefile $(MAKECMDGOALS)
-	$(MAKE) -f MK3-Makefile $(MAKECMDGOALS)
```
[the remark this answers is not in the document]
Comment (scgbckbone) [reply stm32/Makefile:19]: we should not build mk3 with just `make repro` - until this is done https://github.com/coinkite/ck-andrej-issues/issues/4

[the remark this answers is not in the document]
Comment (doc-hex) [reply stm32/Makefile:19]: yeah the other targets are ok, but not 'release' + mk3

File: stm32/repro-build.sh [hunk 2]
```diff
@@ -40,6 +40,7 @@ cd ../stm32
 cd ../releases
 if [ -f *-v$VERSION_STRING-mk$MK_NUM-coldcard.dfu ]; then
     echo "Using existing binary in ../releases, not downloading."
+    PUBLISHED_BIN=`realpath *-v$VERSION_STRING-mk$MK_NUM-coldcard.dfu`
```
[the remark this answers is not in the document]
Comment (scgbckbone) [reply stm32/repro-build.sh:43]: isn't this run from within docker?

**Pull request merged**